sdk/src/client/api/block_builder/input_selection/requirement/mana.rs

```diff
@@ -287,4 +293,39 @@ impl InputSelection {
                 self.creation_slot,
             )?)
     }
+
+    fn sort_available_inputs_for_mana(&mut self, missing_mana: u64) {
+        // Establish the order in which we want to pick an input
+        let sort_order_type = [
+            BasicOutput::KIND,
+            AccountOutput::KIND,
```

Comment:
Do we make sure somewhere that we can actually use the mana amount from the account output? Because of the account bound mana

Reply:
well we really don't get to use the account mana unless we reduce it via the automatic mana allotment process. Otherwise the output is always created with the same amount so we gain nothing.

Reply:
Hm what's the point then of having the account outputs at the second place then?

Reply:
Well technically there's not much point in using anything other than Basic and Account, and accounts are only useful if we're allotting. I could only allow getting those kinds if you think that's better?

Reply:
NFTs can also make sense and have less restrictions than accounts?

Reply:
true